setup.py

```diff
@@ -38,7 +38,7 @@ def package_files(directory):
 
     author='ETH',
 
-    license='GNU General Public License',
+    license='OSI Approved :: GNU General Public License v3 (GPLv3)',
```

Comment:
Official PyPI string for GPLv3: https://pypi.org/search/?q=&o=&c=License+%3A%3A+OSI+Approved+%3A%3A+GNU+General+Public+License+v3+%28GPLv3%29